configs/default/CUST-RPMENERGYF722HD1.config

```
@@ -0,0 +1,115 @@
+# Betaflight / STM32F7X2 (S7X2)
+
+board_name RPMENERGYF722HD1
+manufacturer_id CUST
+
+#define USE_ACCGYRO_BMI270
+#define USE_FLASH_W25N01G
+#define USE_BARO_BMP388
+#define USE_MAG_QMC5883
```

```suggestion
#define USE_ACC
#define USE_BARO
#define USE_FLASH
#define USE_GYRO
#define USE_MAG
#define USE_ACCGYRO_BMI270
#define USE_FLASH_W25N01G
#define USE_BARO_BMP388
#define USE_MAG_QMC5883
```

If magnetometer is not physical present it should be removed.

---

Okay, there will be a magnetometer on the drone, but the magnetometer is not physically on the flight controller so I will remove it from code.